### ISSUE TITLE
Add RQRX?

### ISSUE THREAD

**Issue by grctest** [bitshares/awesome-bitshares#123]
https://rsquared.digital

BTS fork, they're migrating from ETH to their own chain, seems they've got some serializer changes too https://github.com/R-Squared-Project/R-Squared-js/blob/development/lib/serializer/src/operations.js#L1647

**Comment by abitmore** [comment]
Revolution Populi renamed?

**Comment by grctest** [comment]
Yeah, new name and moving to graphene from Eth

**Comment by abitmore** [comment]
In fact Revolution Populi has been listed on awesome-bitshares for a long time. We need to identify if they are the same project or 2 projects.

**Comment by grctest** [comment]
Indeed, it's like a PTS -> BTSX style chain migration. Revolution Populi was closed down, their ethereum contract is closed/halted, with users now able to claim their RQRX through a metamask component in their web ui

**Comment by grctest** [comment]
Resolved by: https://github.com/bitshares/awesome-bitshares/pull/124
